app/models/post.rb

```diff
@@ -0,0 +1,13 @@
+class Post < ApplicationRecord
+  belongs_to :author, class_name: 'User'
+  has_many :comments
+  has_many :likes
+
+  def update_posts_counter
+    author.update(posts_counter: author.posts_counter + 1)
+  end
```

Comment:
- [ ] [OPTIONAL] Please use the [Rails Active Callbacks](https://guides.rubyonrails.org/active_record_callbacks.html) `after_save` to call the `update_posts_counter` method every time a new post has been saved to the database, then make the `update_posts_counter` method private. Kindly do this also for `Like` and `Comment`

Reply:
@gemmen29. thank you very much for your feedback. I will take that into account in my next pull request. thank you once again :+1: